ctapipe/reco/sklearn.py

```diff
@@ -565,7 +881,11 @@ def __call__(self, telescope_type, table):
             random_state=self.rng.integers(0, 2**31 - 1),
         )
 
-        cv_it = kfold.split(table, table[self.model_component.target])
+        if isinstance(self.model_component, DispReconstructor):
+            cv_it = kfold.split(table, np.sign(table[self.model_component.target]))
```

Comment:
Not really sure if stratified kfold on the sign is really required, it should be 50/50 really already by randomly splitting.

Reply:
Can we be sure of that even if a small sample is used for training?